stack.yaml

```diff
@@ -1,12 +1,11 @@
-resolver: lts-13.24
+resolver: nightly-2021-01-05
```

Comment:
It won't build with the latest `lts-16.27`, but it builds and passes all tests on my machine with this nightly resolver.  I imagine this is not ideal (it is also using bleeding edge ghc, which I enjoy, but I guess it i not considered stable... :/).  Regardless, it seems fine as a way to give us the ability to keep using the project right now.

Reply:
I can try building with the `nightly-2020-12-14` resolver too.  It only requires `GHC 8.10.2` instead of `GHC 8.10.3`, but I don't think any earlier snapshots will work at all with the upstream `gi` fixes.

Reply:
Do you know why it doesn't build with the LTS?

I don't have any strong preferences here, I'm not using Stack so I can't say what would be best. But perhaps LTS + a few extra-deps is better than a nightly resolver?

Reply:
Hmm, so I was expanding a project that uses Streamly and Evdev, and I noticed that for that project I needed to use the nightly resolver.  It was much easier for me to add 2 extra-deps that were older.  When I tried to build the project using `LTS` it went to dep hell, because some resources wanted older `gi-*` versions.

It might not matter if this library is built with `LTS` though (I can still try that if we want), but obviously there are big issues with using this along with `lts-16.27`.

At this point it seems to me like we should use the nightly resolver here.  It works the best, and it is a reminder to move to lts-16.x whenever the `gi-*` dependencies are up to date in an lts resolver.  Does that make sense to anyone else?

Reply:
I have not tried it, but might `lts-17` (released a couple of days ago) work?

Reply:
Oh, I hadn't been paying attention to new lts releases. I expect it to work. I will try it out.  thank you!

Reply:
Just to resolve things here.
https://github.com/owickstrom/gi-gtk-declarative/pull/90/commits/fe542ea38437380d00e965963d2880d5c7c169ee
switched to using `lts-17.0`, and it is great.  Thank you for the heads up about the new lts, @Dretch !